Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -664,18 +670,32 @@ def fetch_incidents():
 
     else:
         detections_ids = demisto.get(get_fetch_detections(last_created_timestamp=last_fetch), 'resources')
-    incidents = []
+    incidents = []  # type:List
 
     if detections_ids:
+
+        # make sure we do not fetch the same detection again.
+        if last_detection_id == detections_ids[0]:
+            first_index_to_fetch = 1
+
+            # if this is the only detection - dont fetch.
+            if len(detections_ids) == 1:
+                return incidents
+
+        else:
```

Comment:
add a comment that explains that this is for the first fetch basically

Reply:
it could be more than just the first fetch - I'll add a comment simply saying "if the first detection in this pull is different than the last detection fetched we bring it as well".